### ISSUE TITLE
Umm anything I can do to bring the cursive font clear more readable 

### ISSUE THREAD

**Issue by Droyk** [nagadomi/waifu2x#365]
So here is the original 
![KashmiriChai (1)](https://user-images.githubusercontent.com/15848350/96357703-ebef0a00-111c-11eb-8279-862583fa43cd.png)
after conversion!
![28826ec5-840b-440f-c403-ee4d489ce66b](https://user-images.githubusercontent.com/15848350/96357712-07f2ab80-111d-11eb-8a15-a1ce21304ea9.png)

**Comment by xldsdr124** [comment]
The original image text is not very clear, I think AI may not be possible.

**Comment by nagadomi** [comment]
> I think this is not the purpose of waifu2x.

Yes. But I used Japanese novel mixed with the training data a little. No English curve fonts are used.

**Comment by Droyk** [comment]
@nagadomi any other tool, you guys know that can make this readable! an alternative to waifu2x!
